korman/exporter/material.py

```diff
@@ -518,6 +518,10 @@ def export_dynamic_env(self, bo, layer, texture, pl_class):
             # will probably want to steal it for diabolical purposes...
             pl_env.position = hsVector3(*viewpt.location)
 
+            # We'll also export it as a Root though, where supported (MOUL)
+            root = self._mgr.find_create_key(plSceneObject, bl=viewpt)
+            pl_env.rootNode = root
```

Comment:
why the duplication?

Reply:
Mostly because of the comments indicating a desire for the DCM to use a camera viewpoint

Reply:
Those will require actual cameras to be implemented :wink: